addons/mail/static/src/drag_and_drop/dropzone/dropzone.js

```diff
@@ -0,0 +1,59 @@
+/** @odoo-module **/
+
+import { Component, onMounted, useExternalListener, useRef, useState } from "@odoo/owl";
+
+
+export class Dropzone extends Component {
+    static template = "mail.dropzone";
+    static props = { ref: Object };
```

Comment:
Current decision is to not update the linter (yet?), so instead replace by equivalent ES6 syntax (`Object.assign(Dropzone, {})`).